packages/browser-wallet/src/popup/pages/Account/ManageTokens/TokenList.tsx

```diff
@@ -84,6 +84,10 @@ const InfiniteTokenList = forwardRef<HTMLDivElement, InfiniteTokenListProps>(
     }
 );
 
+function isEven(i: number) {
+    return i % 2 === 0;
+}
```

Comment:
Wait.. I thought `isHex` already did this? Didn't we discuss this at some point?

Reply:
It doesn't seem to. :shrug:

Reply:
It doesn't: (The one that was referenced)
https://github.com/Concordium/concordium-node-sdk-js/blob/64baa0938fbddd813b68fdd91b012769667fe16c/packages/common/src/util.ts#L96